geary-papermc-spawning/src/main/kotlin/com/mineinabyss/geary/papermc/spawning/SpawningFeature.kt

```diff
@@ -79,8 +107,34 @@ class SpawningFeature(context: FeatureContext) : Feature(context) {
                 LocationSpread(spawnPositionReader, triesForNearbyLoc = 10)
             ),
         )
+        val spreadTask = SpreadSpawnTask(db, Bukkit.getWorlds().firstOrNull() ?: error("No worlds found, cannot initialize spread spawning"), spreadConfig)
```

Comment:
Worlds are probably also a good thing to be definable in config, either by reading all WG regions and getting their worlds, or explicitly with a list of worlds in config.